source/Engine.h

```diff
@@ -163,6 +164,8 @@ class Engine {
 		double modifier = 1.;
 	};
 
+	enum MouseButton { Left, Right, Middle };
```

Comment:
Should be an `enum class` to avoid polluting the namespace

Reply:
then put every member on separate lines